packages/components/src/components/input/input.lite.tsx

```diff
@@ -35,12 +34,12 @@ export default function DBInput(props: DBInputProps) {
 	const ref = useRef<HTMLInputElement>(null);
 	// jscpd:ignore-start
 	const state = useStore<DBInputState>({
-		_id: DEFAULT_ID,
-		_messageId: DEFAULT_ID + DEFAULT_MESSAGE_ID_SUFFIX,
-		_validMessageId: DEFAULT_ID + DEFAULT_VALID_MESSAGE_ID_SUFFIX,
-		_invalidMessageId: DEFAULT_ID + DEFAULT_INVALID_MESSAGE_ID_SUFFIX,
-		_descByIds: '',
-		_dataListId: DEFAULT_ID,
+		_id: 'input-' + uuid(),
+		_messageId: this._id + DEFAULT_MESSAGE_ID_SUFFIX,
+		_validMessageId: this._id + DEFAULT_VALID_MESSAGE_ID_SUFFIX,
+		_invalidMessageId: this._id + DEFAULT_INVALID_MESSAGE_ID_SUFFIX,
+		_dataListId: `datalist-` + uuid(),
+		_descByIds: `${this._messageId}`,
```

Comment:
```suggestion
		_descByIds: this._messageId,
```

why couldn't this be a simple assignment?

Reply:
It's a workaround for `vue` output. The ts for vue would think that it could be a Function with out this.